website/src/pages/account/index.tsx

```diff
@@ -2,24 +2,10 @@ import { Button } from "@chakra-ui/react";
 import Head from "next/head";
 import Link from "next/link";
 import { useSession } from "next-auth/react";
-import React, { useState } from "react";
+import React from "react";
 
 export default function Account() {
   const { data: session } = useSession();
-  const [username, setUsername] = useState("null");
-
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  const handleUpdate = async () => {
-    const response = await fetch("../api/update", {
```

Comment:
I double checked and `/api/update` doesn't exist.  It's called `/api/username`.  I'll fix this in a follow up PR where the user explicitly clicks a button to update their username.